website/docs/r/iam_trusted_profiles.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profiles"
+description: |-
+  Manages iam_trusted_profiles.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profiles
+
+Provides a resource for iam_trusted_profiles. This allows iam_trusted_profiles to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_iam_trusted_profiles" "iam_trusted_profiles" {
+  account_id = "account_id"
+  name = "name"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your resource.
+
+* `account_id` - (Required, String) The account ID of the trusted profile.
+* `description` - (Optional, String) The optional description of the trusted profile. The 'description' property is only available if a description was provided during creation of trusted profile.
+* `name` - (Required, String) Name of the trusted profile. The name is checked for uniqueness. Therefore trusted profiles with the same names can not exist in the same account.
```

Comment:
Name of the trusted profile. The name is checked for uniqueness. Therefore trusted profiles with the same names can not exist in the same account.

as

The unique name of the trusted profile. Therefore the trusted profiles with the same names cannot exist in the same account.